template.tpl

```diff
@@ -1942,8 +2167,59 @@ setup: |-
     "didomiVendorsRawConsent": "",
     "didomiVendorsRawConsentDenied": "",
   "didomiVendorsRawConsentUnknown": "didomi:google,c:googleana-4TXnJigR,",
+    "didomiVendorsEnabled": "",
+    "didomiVendorsDisabled": "",
+    "didomiVendorsUnknown": "google,googleana-4TXnJigR,",
   };
 
+  const regionsDeniedByDefault = [
```

Comment:
Is this variable a duplicate of https://github.com/didomi/gtm-template/pull/32/files#diff-dfeab1de57b435942d72fd2eef60b735ad864ee7b35692d09d05d0536ffce292R320?